modules/afmongodb/afmongodb-worker.c

```diff
@@ -397,8 +470,17 @@ _worker_insert(LogThreadedDestWorker *s, LogMessage *msg)
     }
 
   bson_error_t error;
-  success = mongoc_collection_insert(self->coll_obj, MONGOC_INSERT_NONE,
-                                     (const bson_t *)self->bson, NULL, &error);
+  if (owner->mongoc_opt_use_bulk)
+    {
+      if (self->bulk_op == NULL)
+        self->bulk_op = mongoc_collection_create_bulk_operation_with_opts(self->coll_obj, self->bson_opts);
+
+      mongoc_bulk_operation_insert(self->bulk_op, (const bson_t *)self->bson);
+      return LTR_QUEUED;
+    }
+  else
+    success = mongoc_collection_insert(self->coll_obj, MONGOC_INSERT_NONE,
```

Comment:
We should also pass write-concern options to `mongoc_collection_insert()` to support acked/unacked/wmajority in non-bulk mode as well.

Reply:
good point, missed that
done